src/commands/createNewProject/createNewProject.ts

```diff
@@ -106,3 +110,25 @@ export function getProjectCreator(language: string, functionAppPath: string, act
             return new ScriptProjectCreatorBase(functionAppPath, actionContext.properties);
     }
 }
+
+async function addToFilesExclude(filesToExclude: string | string[], fsPath: string): Promise<void> {
+    const projectConfiguration: WorkspaceConfiguration = workspace.getConfiguration('files', Uri.file(fsPath));
+    // tslint:disable:no-any no-unsafe-any
+    let excludedFiles: any | undefined = projectConfiguration.get('exclude');
```

Comment:
Intellisense shows that vscode has 'files.exclude' typed to 'object' so you could use {}|undefined

Reply:
If I use the `{}` type, it will throw a type definition warning that `property` does not exist (obj or bin in this case)

Reply:
Then you should more strongly define it: {bin?: boolean, obj?: boolean}.

Also makes interpreting the code easier.